.pre-commit-config.yaml

```diff
@@ -34,3 +34,4 @@ repos:
     hooks:
       - id: prettier
         args: ["--print-width", "120"]
+        require_serial: true
```

Comment:
@benoit9126 this turned out to be the reason prettier was very slow in pre-commit.
`pre-commit run prettier -a` now runs in about 1.2 seconds compared to 8 -> 16 seconds without this option.